### PR TITLE
Make it 3.4 compatible

### PR DESCRIPTION
**Pull request by dereuromark**
We now need this in a release in order to not have to use dev-dev for Cake3.4.

### CODE REVIEW

File: src/Auth/HybridAuthAuthenticate.php [hunk 1]
```diff
@@ -422,4 +431,25 @@ public function implementedEvents()
     {
         return ['Auth.logout' => 'logout'];
     }
+
+    /**
+     * @param string|array $url URL
```
Comment (ADmad) [inline src/Auth/HybridAuthAuthenticate.php]: Please add description.

File: src/Auth/HybridAuthAuthenticate.php [hunk 2]
```diff
@@ -422,4 +431,25 @@ public function implementedEvents()
...
+     * @param string $redirectQueryString Redirect query string
+     * @return string URL
+     */
+    protected function appendRedirectQueryString($url, $redirectQueryString)
```
Comment (ADmad) [inline src/Auth/HybridAuthAuthenticate.php]: Prefix the method with underscore, I prefer maintaining consistency with other protected methods.

**Comment by dereuromark** [comment]
I let you finish this up, it is in your repo anyway (I dont have access rights).
Thanks for the release in advance.

**Comment by ADmad** [comment]
@dereuromark I have merged the changes to master. Please just try it once and if everything is hunky dory I'll do a release.

**Pull request merged**